### PR TITLE
Added s-spaced-words

### LINKED ISSUE #97: Add s-space-words?
Similar to `s-dash-words`, `s-snake-case` etc. but returning space separated words.

Implementation could be something like

```
(s-join " " (s-split-words arg))
```

### PR DESCRIPTION
**Pull request by jasonm23**
Resolves #97

### CODE REVIEW

**Comment by jasonm23** [comment]
@magnars 👍🏽 👎🏽 ?

**Comment by magnars** [comment]
Makes sense 👍

File: s.el [hunk 1]
```diff
@@ -233,9 +233,9 @@ When not specified, ELLIPSIS defaults to ‘...’."
   (declare (pure t) (side-effect-free t))
   (let ((extra (max 0 (- len (length s)))))
     (concat
-     (make-string (ceiling extra 2) ? )
+     (make-string (ceiling extra 2) ?)
```
Comment (tarsius) [inline s.el:236]: `(? )` is an unfortunate way of writing a space character (inside a list).
`(?)` is a syntax error.

I recommend you use `(?\s)`

Comment (jasonm23) [reply s.el:236]: Nice catch.

**Review by jasonm23 — COMMENTED** [review]
Tests running green

**Pull request merged**